File: Programação/Cursos/Lógica de Programação/Python/Exercicio03.py
import math as mt
altura = float(input("Dite a altura do retangulo: "))
largura = float(input("Dite a largura do retangulo: "))
print(f"Area: {altura * largura}")
print(f"Perimetro: {altura * 2 + largura * 2}")
print(f"Diagonal: {mt.sqrt(altura ** 2 + largura ** 2)}")

File: Programação/Cursos/Lógica de Programação/C/Exercicio03.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>

int main (){
    float altura, largura;
    printf("Digite a altura do retangulo: ");
    scanf("%f", &altura);
    printf("Digite a largura do retangulo: ");
    scanf("%f", &largura);
    printf("Area = %.2f\nPerimetro = %.2f\n", altura * largura, (altura * 2) + (largura * 2));
}